.scrutinizer.yml

```diff
@@ -0,0 +1,15 @@
+build:
+    environment:
+        php:
+            version: 7.1
```

Comment:
Arent those files 2 space indentation? 
Please also check if .editorconfig is set correctly.